src/hero.tsx

```diff
@@ -138,7 +138,9 @@ const Hero = React.forwardRef<HTMLDivElement, HeroProps>(
         ref
     ) => {
         const {isTabletOrSmaller} = useScreenSize();
-        const isInsideSlideShow = useIsInsideSlideshowContext();
+        const slideshowContextValue = useSlideshowContext();
+        const isInsideSlideShow = !!slideshowContextValue;
```

Comment:
We won't need this variable `isInsideSlideshow` after we merge https://github.com/Telefonica/mistica-web/pull/1100